packages/dev/core/src/Misc/computePressure.ts

```diff
@@ -20,24 +20,30 @@ export class ComputePressureObserverWrapper {
      * Returns true if ComputePressureObserver is available for use, false otherwise.
      */
     public static get IsAvailable() {

```

Comment:
I also considered adding a `source` argument to `IsAvailable` but that isn't possible since it is a getter. It would then check for that source with `includes(source)` instead of `includes("cpu")`. Let me know if you prefer that and I can make it a regular function.

Reply:
We cannot change that getter because of backward compat but we can do this:
- isAvailable tests on cpu
- we add a new function to test for a specific source

Reply:
Ok, makes sense. Maybe it isn't necessary for this change.